README.md

```diff
@@ -2398,8 +2400,87 @@ hazelcastClient.getMap('brothersMap').then(function (mp) {
 });
 ```
 
-## 7.9. Monitoring and Logging
+## 7.8. Performance
+
+### 7.8.1. Partition Aware
+
+Partition Aware ensures that related entries exist on the same member. If related data is on the same member, operations can be executed without the cost of extra network calls and extra wire data and this improves the performance. This feature is provided by using the same partition keys for related data.
+
+Hazelcast has a standard way of finding out which member owns/manages each key object. The following operations will be routed to the same member, since all of them are operating based on the same key `'key1'`.
+
+```javascript
+Client.newHazelcastClient().then(function (client) {
+    hazelcastClient = client;
+    return hazelcastClient.getMap('mapA')
+}).then(function (mp) {
+    mapA = mp;
+    return hazelcastClient.getMap('mapB');
+}).then(function (mp) {
+    mapB = mp;
+    return hazelcastClient.getMap('mapC');
+}).then(function (mp) {
+    mapC = mp;
 
+    // since map names are different, operation will be manipulating
+    // different entries, but the operation will take place on the
+    // same member since the keys ('key1') are the same
+    return mapA.put('key1', 'Furkan');
+}).then(function () {
+    return mapB.get('key1');
+}).then(function (res) {
+    return mapC.remove('key1');
+}).then(function () {
+    // lock operation will still execute on the same member
+    // of the cluster since the key ("key1") is same
+    return hazelcastClient.getLock('key1');
+}).then(function (l) {
+    lock = l;
+    return lock.lock();
+});
+```
+
+When the keys are the same, entries are stored on the same member. However, we sometimes want to have related entries stored on the same member, such as a customer and his/her order entries. We would have a customers map with `customerId` as the key and an orders map with `orderId` as the key. Since `customerId` and `orderId` are different keys, a customer and his/her orders may fall into different members in your cluster. So how can we have them stored on the same member? We create an affinity between customer and orders. If we make them part of the same partition then these entries will be co-located. We achieve this by making `orderId`s `PartitionAware`.
```

Comment:
"by making `OrderKey`s `PartitionAware`" here

Reply:
It should also be changed in Hazelcast IMDG Reference Manual.